src/pages/home/report/ReportActionItem.js

```diff
@@ -538,12 +536,10 @@ function ReportActionItem(props) {
             }
 
             return (
-                <OfflineWithFeedback pendingAction={props.action.pendingAction}>
```

Comment:
Are you 100% confident that this is the same case? In the case of `MoneyRequestView`, you observed that the removed pending action `props.action.pendingAction` is equivalent to the existing conditions `|| lodashGet(transaction, 'pendingAction')}`. I cannot see such a pattern in the case of `TaskView`.

I agree that it's great to fix another code fragment if it has an issue with the same root cause, but we're also responsible for all potential regressions we introduce.

Reply:
Is it different from how it works in case of `IOU` actions?

What I'm trying to figure out is why the pattern of checking for pendingActions is different in `TaskView` and `MoneyRequestView`, i.e. that in `MoneyRequestView` we always check for either the outer pending action _or_ for the field-level pending action, and in `TaskView` we only check for the field-level pending actions.

Reply:
Both `IOU` and `Task` actions have a similar set of operations, so I can't see a reason for these patterns to differ. But maybe we're exceeding the scope of this issue now...

Still, I think that if we're aware of these possible different combinations of pending actions (field-level vs the outer pending action), we should cover them in tests. Would you be able to describe the tests which ensure that the discussed `||` alternative works as expected after the PR changes?

Reply:
And, if possible, tests that affect tasks fields separately, demonstrating that such an alternative check is not needed, and we didn't break anything by removing the outer `OfflineWithFeedback` wrapper for tasks.

Reply:
Looks fine 👍